test/ir/pir/translator/test_push_dense_translator.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2024 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import test_op_translator
+
+import paddle
+from paddle.base.layer_helper import LayerHelper
+
+
+class TestPushDenseOpTranslator(test_op_translator.TestOpTranslator):
+    def append_op(self):
+        self.op_type = "push_dense"
+        ids = paddle.ones(shape=(100, 2, 3), dtype='float32')
+        input_names = []
+        attrs = {
+            'table_id': 1,
```

Comment:
```suggestion
            'TableId': 1,
```
这里旧IR下Attribute的命名采用的是驼峰命名，所以这里对旧IR组网要使用驼峰命名。新IR下使用的是下划线方式，这里同时也需要在op_compat里面配置一下attr的名称映射。

Reply:
谢谢！之前一直看不明白为什么它判断不是int32